serenata_toolbox/session_start_times.py

```diff
@@ -0,0 +1,57 @@
+import urllib
+import xml.etree.ElementTree as ET
+
+import pandas as pd
+
+from serenata_toolbox import datasets
+from serenata_toolbox.cleanup import xml_extract_text, \
+                                     xml_extract_datetime, \
+                                     translate_column
+
+class SessionStartTimes:
+    URL = (
+        'http://www.camara.leg.br/SitCamaraWS/sessoesreunioes.asmx/ListarPresencasDia'
+        '?siglaPartido=&siglaUF='
+        '&data={0}'
+        '&numMatriculaParlamentar={1}'
+    )
+
+    def fetch(self, pivot, session_dates):
+        """
+        :param pivot: (int) a congressperson document to use as a pivot for scraping the data
+        :param session_dates: (str) a list of datetime objects to fetch the start times for
```

Comment:
I'd use use `:param session_dates: (list) datetime objects to fetch the start times for…`